src/near-rpc-provider.ts

```diff
@@ -354,7 +356,7 @@ export class NearRpcProvider extends JsonRpcProvider {
     }
   }
 
-  async getContractState(addressOrName: string, blockTag: BlockTag) {
+  async getContractState(addressOrName: string, blockTag: BlockTag): Promise<GetStateResponse> {
```

Comment:
similar to above, let's keep the input types consistent so let's allow Promises here as well and `await` them in our code

Reply:
No problem!